exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/export_as_is.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>... you <strong>will</strong> learn (just for the sake of consistency)</p><p> </p><p>The base flavor <code>template-Exasol-all-python-3.10</code> is <strong>a </strong>part of the Script-Languages Release repository and it's purpose is to be the base for customer modifications which aim to use Python UDF's with Python packages which are not included in the Standard Container of the Exasol database. </p><p> </p><p>This is a very long sentence. I suggest breaking it into several parts. How about this version: "The base flavor template-Exasol-all-python-3.10 is a part of the Script-Languages Release repository. It serves as a base for a customized Language Container. The latter is required when user-provided Python UDF's depend on packages not included in the Standard Container of the Exasol database."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>fixed</p>